mautic/mautic
# Fix empty slots in builder

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5749
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This is follow up to https://github.com/mautic/mautic/pull/6454 based on this issue https://github.com/mautic/mautic/issues/5749
If  you leave your slots empty, then you can't edit it again. 
This PR added placeholder If slot is empty, then you can edit it.

![image](https://user-images.githubusercontent.com/462477/47960977-dd2bed00-e002-11e8-81bc-94b5a0a0cbd1.png)

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create email and open builder
2. Add text slot  and remove default text
3. You can edit slot 

#### Steps to test this PR:
1. Repeat all steps
2. See If you can edit slot without content
You can test text, dynamic content and code mode slot.
